tests/services/test_criteria_matcher.py

```
@@ -12,49 +12,33 @@
-    return _mock_stat
-
-
-@pytest.fixture
-def mock_file(tmp_path, mocker):
-    """Fixture to create a mock file and optionally mock stat."""
+def real_file(tmp_path):
+    """Fixture to create a real file with specific attributes."""
 
     @contextlib.contextmanager
-    def _mock_file(filename="test.txt", *, create_file=True, stat_info=None):
+    def _real_file(filename="test.txt", size=0, mtime=None, atime=None, content=None):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_real_file`

(ANN202)